app/menu.tsx

```diff
@@ -26,9 +25,8 @@ type NavigationState = {
     Menu: undefined;
 };
 
-export const MenuScreen = ({
-    navigation,
-}: NativeStackScreenProps<NavigationState, 'Menu'>) => {
+//TODO use a proper type
+export const MenuScreen = ({ navigation }: any) => {
     const { width } = useWindowDimensions();
 
     const dispatch = useDispatch<AppDispatch>();
```

Comment:
Nice to see that you're using Redux. It should make leveraging the web client code easier.